### DISCUSSION TITLE
各位绳匠，绳网因为有法律风险而将要关闭，之后我们应该何去何从

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#5500] [公告]
😭回来吧我的绳网😭
🌟我最骄傲的信仰🌟
⚡️历历在目的教程⚡️
😭眼泪莫名在流淌😭
💥依稀记得钓鱼佬💥
👍还有给力的绳斤👍
⚡️把天马速递打退⚡️
✨通宵熬夜都不累✨
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/651fba00ba55004182fffbb5aa553c53_6086077287436441482.png" alt="米仔_膜拜" width="75" >

**Comment by fzdqz** [comment]
回米游社尝试重建绳网🙃

**Comment by fzdqz** [comment]
等等<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >我们钓鱼佬怎么也在这上面
